src/Accessor/DefaultAccessorStrategy.php

```diff
@@ -70,22 +70,25 @@ public function getValue(object $object, PropertyMetadata $metadata, Serializati
 
         if (null === $metadata->getter) {
             if (!isset($this->readAccessors[$metadata->class])) {
-                if (true === $metadata->forceReflectionAccess) {
```

Comment:
`forceReflectionAccess` was meant for that propouse (and calculated at metadata-discover time), I understand that this might work but the propose of that is to avoid checking  `$property->getDeclaringClass()->isInternal()` for each property access

Reply:
What's the solution, caching?
Can't metadata provide this flag?

Reply:
This line: https://github.com/schmittjoh/serializer/blob/88be308c0ab5a75fae631ccff3188ba68de8a428/src/Metadata/PropertyMetadata.php#L145 should be doing exactly so

Reply:
Done